terminal.lua

```diff
@@ -1,7 +1,7 @@
 -- This is a VT100 terminal output writer for Pandoc.
 -- Inwoke with: pandoc -t terminal.lua
 
--- Copyright (c) 2018 Orange
+-- Copyright (c) 2018 — 2020 Orange
 -- Homepage: https://github.com/Orange-OpenSource/pandoc-terminal-writer
 -- This module is released under the MIT License (MIT).
 -- Please see LICENCE.txt for details.
```

Comment:
you should definitely add yourself to the authors list here

Reply:
Done online (nice GitHub feature, BTW)